### ISSUE TITLE
thr result of your pytorch version of the code on the coco validation set

### ISSUE THREAD

**Issue by superWSL** [caiyuanhao1998/RSN#15]
Can you tell me the highest result of your pytorch version of the code on the coco validation set

**Comment by caiyuanhao1998** [comment]
Hi, please refer to our paper for the highest results :)
The highest results are achieved by the Megvii inner deep learning framework, MegDL
